src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const Tac = () => (
     <div>
         <h2 data-anchor>{it.L('Terms and Conditions')}</h2>
-        <p><i>{it.L('Version 44. Last modified 2018-04-10.')}</i></p>
+        <p><i>{it.L('Date effective 2018-04-10.')}</i></p>
```

Comment:
how about the wording here? https://github.com/binary-com/binary-static/blob/master/src/templates/static/legal/metatrader/terms_and_conditions.jsx#L6 does that also need to be updated?

Reply:
There's been no change to MT5, but we can change it if you want the language consistent.